src/test/osd/osd-scrub-snaps.sh

```diff
@@ -17,6 +17,15 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    DIFFCOLOPTS=""
+else
+    termwidth=$(stty -a | head -1 | sed -e 's/.*columns \([0-9]*\).*/\1/')
+    if test -n "$termwidth" -a "$termwidth" != "0"; then termwidth="-W ${termwidth}"; fi
+    DIFCOLOPTS="-y -W $termwidth"
```

Comment:
@dzafman 
David, I missed out on your comment here.
I'll address this over the weekend as it is a good idea.